src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+
+  if (days >= 3 && days < 7) {
```

Comment:
[checklist]
Don't use literals in expressions (HARDCODE). Instead, use constants explaining this value.

for example,
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;

if (numberOfDays >= LONG_TERM) {
  return basePrice - LONG_TERM_DISCOUNT;
}